src/orm/generateORM.ts

```diff
@@ -1,4 +1,11 @@
-import { PropertiesTypes, IOrmCatalog, PycswLayerCatalogRecord, Pycsw3DCatalogRecord, ICatalogDBEntityMapping } from '@map-colonies/mc-model-types';
+import {
+  PropertiesTypes,
+  IOrmCatalog,
+  PycswLayerCatalogRecord,
+  Pycsw3DCatalogRecord,
+  ICatalogDBEntityMapping,
+  SpecialORMColumnType,
```

Comment:
did not see **Special**ORMColumnType
ORMColumnType is an enum

Reply:
fixed